src/ui/UIBase.js

```diff
@@ -488,6 +488,7 @@ class UIElement extends BaseUIElement {
         return erasedChild;
     }
 
+    // TODO deprecate?
```

Comment:
It's not used anywhere in the codebase so you can delete it if you feel like it.